cli/yarn.lock

```diff
@@ -1144,11 +1144,6 @@ hoopy@^0.1.4:
   resolved "https://registry.yarnpkg.com/hoopy/-/hoopy-0.1.4.tgz#609207d661100033a9a9402ad3dea677381c1b1d"
   integrity sha512-HRcs+2mr52W0K+x8RzcLzuPPmVIKMSv97RGHy0Ea9y/mpcaK+xTrjICA04KAHi4GRzxliNqNJEFYWHghy3rSfQ==
 
-https@^1.0.0:
```

Comment:
WTF yarn

I didn't touch any dependencies, I just did `yarn build` in a devcontainer, after making my changes. Let's confirm if we really don't need t.

Reply:
we don't maybe it's a remnant from a yarn.lock file, I briefly used it in the cli and kicked it out again